tommath.def

```diff
@@ -23,10 +23,10 @@ EXPORTS
     mp_copy
     mp_count_bits
     mp_decr
+    mp_div
```

Comment:
file looks the same except for the sorting, perl sorts differently than sort command it seems. sorry for that.